### PR TITLE
REL-2231: Display full PIT version in compatibility error message

### PR DESCRIPTION
**Pull request by cquiroz**
Some small changes to the way the error message is presented to the user if the backends are incompatible with the PIT version

### CODE REVIEW

File: bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/submit/SubmitStatus.scala [hunk 1]
```diff
@@ -41,12 +41,16 @@ object SubmitStatus {
 
   def submissionClosed(destination: Option[SubmitDestination]) = s"The ${destinationName(destination)} proposal server is currently closed. Please see the Gemini web pages for proposal submission dates"
 
-  private val ErrorRegex = """.* (\d\d\d\d.) ([\d\.]*).*""".r // Extract semester and version from the backend error message
+  private val VersionMismatchRegex = """.* (\d\d\d\d)([A|B]) \d{4}(\.[\d\.]*).*""".r // Extract semester and version from the backend error message
+
+  def parseServiceError405(destination: Option[SubmitDestination], message: String) = {
+    val VersionMismatchRegex(year, sem, version) = message
```
Comment (tpolecat) [inline bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/submit/SubmitStatus.scala]: If this doesn't match you get a `MatchError` ... do you need to handle this case?

Comment (cquiroz) [inline bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/submit/SubmitStatus.scala]: Good point, this assume the backend is well behaved, which is not a 100% solid assumption

**Comment by cquiroz** [comment]
Fixed to consider @tpolecat comment

**Comment by tpolecat** [comment]
:+1:

**Pull request merged**